health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -176,51 +180,49 @@ export const UICustomizations = {
     },
 
-
-
-
     additionalCustomizations: (row, key, column, value, t, searchResult) => {
-      
-      
       if (key === "Role") {
-        
-        if (value && value !== "NA") {
+        debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)